src/services/work.js

```diff
@@ -127,6 +127,19 @@ export const getDetails = (work) => {
   return formData;
 };
 
+export const getSolutionsCount = async (workId) => {
+  const response = await axios.get(
+    `${config.API.V5}/submissions?challengeId=${workId}&perPage=500`
+  );
+  const submissions = response.data;
+  return submissions.length;
+  
+  // const response = await axios.head(
```

Comment:
Hey @brooketopcoder @testflyjets Please note this commented code is the right way to implement this. I'll just uncomment it and remove the above "workaround" when the api is fixed.